mindjunkies/payments/views.py

```
@@ -131,6 +136,27 @@
             )
 
+            # BalanceHistory.objects.create(user=user, transaction)
+            teacher = course.teacher
+            transaction = Transaction.objects.get(tran_id=data["tran_id"])
+            prev_balance = teacher.balance.amount
+            teacher.balance.amount += course.course_price
+            teacher.balance.save()
+
+            new_balance = teacher.balance.amount
+
+            
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)